### ISSUE TITLE
2FA

### ISSUE THREAD

**Issue by mpchadwick** [readthedocs/ethical-ad-server#305]
A good enhancement would be to add the ability to secure accounts with 2FA

**Comment by davidfischer** [comment]
Agreed. It's something we've wanted here and on the [readthedocs.org](https://github.com/readthedocs/readthedocs.org) codebase as well. There isn't "one way" to do this with Django but maybe it's time to just bite the bullet and implement something over waiting for a better solution.
